app/product.php

```diff
@@ -52,5 +52,16 @@ public function findProduct($id)
         return $findProduct;
     }
 
+    function rand_string($length)
```

Comment:
This function should probably be in a Helper class:
https://laravel.com/docs/5.4/helpers